db/migrate/20230415003243_create_courses.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCourses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :courses do |t|
+      t.string :name
+      t.string :directory_path
+      t.text :info
+      t.boolean :private, default:false
+      t.integer :instructor_id
```

Comment:
To ensure accurate data entry, it is recommended to utilize references instead of directly adding an instructor ID.
```ruby
add_reference :courses, :instructor, foreign_key: { to_table: :users }, null: false

```

Reply:
We have created the foreign key through another migration add_index_to _courses.